### DISCUSSION TITLE
How to manually import the rules?

### DISCUSSION THREAD

**Discussion by thibaultmol** [OhMyGuus/I-Still-Dont-Care-About-Cookies#15765] [Q&A]
I see that monketype.com is for example been added through a pull requets but it doesn't work with the extension currently. so i'm assuming the extension somehow is using an old ruleset?

**Comment by erik55** [comment]
You can clone the repo. Then in Firefox you can go to <about:debugging>,  then select *This Firefox*, then click on *Load temporary addon* button, there go to the `src` folder of the project, then click on the `manifest.json` (I had to rename or create a link from the `manifest_v2.json` to `manifest.json`). It should now work.
